app/lib/ui/components/component.dart

```diff
@@ -1,74 +1,60 @@
 import 'dart:core';
 
+import 'package:flutter/cupertino.dart';
+
+import 'cards/component_cards.dart';
+import 'checkboxes/component_checkboxes.dart';
+import 'material/component_material.dart';
+import 'radio_buttons/component_radio_buttons.dart';
+import 'switches/component_switches.dart';
+
 class Component {
   String titleRes;
   String imageRes;
   int smallImageRes;
```

Comment:
This property is not used and always set to `1`, I think we can remove it for now.